packages/util/test/unit.spec.ts

```diff
@@ -0,0 +1,30 @@
+import tape from 'tape'
+import { Units } from '../src'
+
+tape('unit', function (t) {
+  t.test('should work for big unit to small unit', function (st) {
+    st.equal(Units.convert('1', 'ether', 'wei'), '1000000000000000000')
```

Comment:
once you have the enum, then these can be rewritten e.g.

```suggestion
    st.equal(Units.convert('1', Unit.Ether, Unit.Wei), '1000000000000000000')
```

Reply:
can you also test with the other input value types other than string to ensure the resulting values are the same, so number and bigint (whichever types are accepted in the method)